Remotion/ObjectBinding/Web/Res/Themes/NovaViso/HTML/BocList.css

```diff
@@ -127,6 +127,25 @@ div.bocList table.bocListTable td.bocListTitleCell > span
   padding: calc(var(--remotion-themed-spacing) / 2) var(--remotion-themed-spacing);
 }
 
+div.bocList table.bocListTable th.bocListTitleCell > a > img,
+div.bocList table.bocListTable th.bocListTitleCell > span > img
+{
+  width: 16px;
+  height: 16px;
+  margin-right: var(--remotion-themed-spacing); /* Provides a spacing between icon and text, which is reset below if only an icon is displayed */
+}
+
+div.bocList table.bocListTable th.bocListTitleCell > a > img:has(+ span.screenReaderText),
+div.bocList table.bocListTable th.bocListTitleCell > span > img:has(+ span.screenReaderText)
+{
+  margin-right: 0;
+}
+
+div.bocList table.bocListTable th.bocListTitleCell > span > img
+{
+  margin: 1px; /* Prevent layout changes due to a missing 1px border when switching between sortable/non-sortable. */
```

Comment:
not sure I understand this. Yes, the span doesn't have a border-placeholder applied. Is this a gneeral layout bug and we need to apply the missing placeholder-border to the span?

either way, please update to use `--remotion-themed-border-width` instead of magic number.

For cases such as this, I actually have a border placeholder defined:
`border: var(--remotion-themed-border-placeholder)`
This is applied when no border is visible. Unless there's a good reason, we should go with this option.

So, to sumarize:
```
div.bocList table.bocListTable th.bocListTitleCell > span,
div.bocList table.bocListTable td.bocListTitleCell > span
{
  border:  var(--remotion-themed-border-placeholder);
}
```
(we need to do both `td` and `th`.)